scripts/pre-commit.sh

```diff
@@ -55,14 +53,14 @@ function launch-minikube-cluster() {
 
 function run-e2e-tests() {
   ${E2E_TEST_CMD}
-  rc=$((rc || $?))
-  return ${rc}
 }
 
 function run-namespaced-e2e-tests() {
-  ${E2E_TEST_CMD} -federation-namespace=foo -registry-namespace=foo -limited-scope=true
-  rc=$((rc || $?))
-  return ${rc}
+  local namespaced_e2e_test_cmd="${E2E_TEST_CMD} -federation-namespace=foo -registry-namespace=foo -limited-scope=true"
+  # Run the placement test separately to avoid crud failures if
+  # teardown doesn't remove namespace placement.
```

Comment:
The `Placement` test explicitly removes the namespace placement and fails the test if it doesn't remove it. Do you know why teardown isn't removing the namespace placement?

Reply:
Ah ok. I assumed this PR was reducing some flakes in e2e testing.